hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -150,39 +152,25 @@ public void write(byte[] b, int off, int len) throws IOException {
         || ((off + len) < 0)) {
       throw new IndexOutOfBoundsException();
     }
-    int rem = len;
-    while (rem > 0) {
-      try {
-        blockOutputStreamEntryPool.allocateBlockIfNeeded();
-        int currentStreamIdx = blockOutputStreamEntryPool
-            .getCurrentStreamEntry().getCurrentStreamIdx();
-        int bufferRem =
-            ecChunkBufferCache.dataBuffers[currentStreamIdx].remaining();
-        int writeLen = Math.min(rem, Math.min(bufferRem, ecChunkSize));
-        int pos = handleDataWrite(currentStreamIdx, b, off, writeLen);
-        checkAndWriteParityCells(pos);
-        rem -= writeLen;
-        off += writeLen;
-      } catch (Exception e) {
-        markStreamClosed();
-        throw new IOException(e.getMessage());
+    try {
+      int i = 0;
```

Comment:
Let's change i as "writtenLen" ?

Also change the below "i+off" to "off + writtenLen"